### PR TITLE
Consistent 4.0 changelog docs formatting

### PR DESCRIPTION
**Pull request by chillu**
Added diff styling, requires merge of https://github.com/silverstripe/silverstripe-docsviewer/pull/134.

### CODE REVIEW

File: docs/en/04_Changelogs/4.0.0.md [hunk 1]
```diff
@@ -462,67 +459,69 @@ The below demonstrates how you can provide new localisation strings for an objec
...
 
 
-    :::php
-    public function pluralise($count)
-    {
-        return _t('MyObject.PLURALS', 'An object|{count} objects', [ 'count' => $count ]);
-    }
+```php
+public function pluralise($count) {
+    return _t('MyObject.PLURALS', 'An object|{count} objects', [ 'count' => $count ]);
```
Comment (unclecheese) [inline docs/en/04_Changelogs/4.0.0.md]: PSR-2 issue

File: docs/en/04_Changelogs/4.0.0.md [hunk 2]
```diff
@@ -1222,36 +1196,37 @@ See our ["Caching" docs](/developer-guides/performance/caching) for more details
...
-            'servers' => array(
-                'host' => 'localhost', 
-                'port' => 11211, 
-            )
+```php
+Cache::add_backend(
+    'primary_memcached', 
+    'Memcached',
+    array(
+        'servers' => array(
```
Comment (unclecheese) [inline docs/en/04_Changelogs/4.0.0.md]: You seem to be enforcing short array syntax (every other one of your updates used it), so might be good to have consistency here.

**Comment by chillu** [comment]
@unclecheese I've fixed all `array` use as well as that one PSR oversight :)

**Comment by chillu** [comment]
Given this is a docs PR, and Aaron has approved it, I'm merging without waiting for the builds to pass

**Comment by unclecheese** [comment]
The build had actually failed, FWIW. I had rebased it and pushed back up to see if that was the cause.

**Pull request merged**